src/protocol/thread/thread.ts

```diff
@@ -1168,7 +1168,7 @@ class _ThreadFront {
         return false;
       }
       kind = minifiedInfo.kind;
-      assert(kind != "prettyPrinted", "source kind must be prettyPrinted");
+      assert(kind != "prettyPrinted", "source kind must not be prettyPrinted");
```

Comment:
Non-blocking nit: move to `!==`, but this pattern of eqeq checks seems to be present in more than one place.